stack/database.py

```diff
@@ -117,8 +144,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion=Ref(db_engine_version),
+    MultiAZ=Ref(db_multi_az),
```

Comment:
Even if you have a MultiAZ instance, the backup is only used for failover. It doesn't even support reads, let alone writes. The purpose for making it configurable is that MultiAZ isn't really need for staging sites but it's a good idea for production sites.